ibexa/commerce/5.0/config/packages/security.yaml

```
@@ -83,8 +83,8 @@ security:
         # Uncomment `ibexa_cdp` rule if you are using Ibexa CDP connector
         #ibexa_cdp:
         #    pattern: /cdp/webhook
```

I didn't notice it before, but I assume this pattern should only match if route **ends** with `/cdp/webhook`?
```suggestion
        #    pattern: /cdp/webhook$
```

---

As discussed internally, brought back unused `Ibexa\Cdp\Security\RequestMatcher` and made sure it's being used, ref: https://github.com/ibexa/recipes-dev/pull/136/commits/8c928fab22ef08fcca28617655e25e47d22da339.